map/views.py

```diff
@@ -1299,19 +1299,18 @@ def get_tile(self, req, ver, zoom, x, y,
             img = Image.open(tilefn)
 
             ra, dec = wcs.crval
+            img_cx, img_cy = wcs.crpix
             pixscale = wcs.pixel_scale()
 
-            width, height = img.size
-
-            ralo = ra - ((width / 2) * pixscale / 3600 / np.cos(np.deg2rad(dec)))
-            rahi = ra + ((width / 2) * pixscale / 3600 / np.cos(np.deg2rad(dec)))
-            declo = dec - ((height / 2) * pixscale / 3600)
-            dechi = dec + ((height / 2) * pixscale / 3600)
+            ralo = ra - (img_cx * pixscale / 3600 / np.cos(np.deg2rad(dec)))
+            rahi = ra + (img_cx * pixscale / 3600 / np.cos(np.deg2rad(dec)))
+            declo = dec - (img_cy * pixscale / 3600)
+            dechi = dec + (img_cy * pixscale / 3600)
 
```

Comment:
I don't think this change is necessary -- wcs.crpix is not guaranteed to be in the middle of the image.

Actually, instead of ra,dec = wcs.crval, you should do
```
ra,dec = wcs.radec_center()
```